motr/ut/idx_dix.c

```diff
@@ -1140,7 +1140,7 @@ static void dtm0_ut_send_redo(const struct m0_fid *ifid,
 	 * Ivan Alekhin.
 	 */
 	struct m0_fom           zero_fom_to_be_deleted = {};
-	/* Extreme hack to convert index fid to component catalogue fid. */
+	/* Extreme method to convert index fid to component catalogue fid. */
```

Comment:
Again, this (and the next one) sounds very forced. What is the reason to get rid of "hack"? I checked a few dictionaries and none of them records any offensive meaning of this word.

Reply:
Reverted to the original word, it is going to be added in the exceptions list.